### DISCUSSION TITLE
[Windows] Error 'Type universe cannot resolve assembly: Uno.UI' after updating VS 17.12.0 Preview 2.1

### DISCUSSION THREAD

**Discussion by mcNets** [unoplatform/uno#18305] [Q&A]
I'm getting this error just after updating my VS to 17.12.0 Preview 2.1 if I try to build Windows project.

"Uno.Sdk": "5.5.0-dev.46"

```
2>XamlCompiler error WMC1006: Cannot resolve Assembly or Windows Metadata file 'Type universe cannot resolve assembly: Uno.UI, Version=255.255.255.255, Culture=neutral, PublicKeyToken=null.'
2>Done building project "OsmoManager.csproj".
```

Desktop project builds and run.

**Comment by jeromelaban** [comment]
I'm not reproducing this issue. It looks like you have some specific dependency that is causing this particular issue. If you can provide a binlog, it will help.

**Reply by mcNets** [reply]
@jeromelaban  Yes, there is an additional dependency. (It is the app I'm using to test Nick's tool.)

**I've tried with another project and it builds.**
